ddtrace/propagation/http.py

```diff
@@ -116,7 +116,7 @@ def _dd_id_to_b3_id(dd_id):
     """Helper to convert Datadog trace/span int ids into hex ids"""
     # DEV: `hex(dd_id)` will give us `0xDEADBEEF`
     # DEV: this gives us lowercase hex, which is what we want
-    return "{:x}".format(dd_id)
+    return "{:016x}".format(dd_id)
```

Comment:
thought for potential future refactor: would it be possible to move this b3-specific logic out of this generic module and into e.g. the b3 integration?